### ISSUE TITLE
[PvP-Progression] Daily Random Battleground and amount of player requirements

### ISSUE THREAD

**Issue by balleny** [chromiecraft/chromiecraft#5983]
### What client do you play on?

enUS

### Faction

Both

### Content Phase:

Generic

### Current Behaviour

Currently, battlegrounds start intentionally with fewer people required than fitting into the battleground.

Further, with the few amount of people required to start a battleground and the currently existing bug that people arent assigned teams evenly at the beginning, many e.g. warsong games start as 3v1 or 4v2

### Expected Blizzlike Behaviour

Blizzlike, a battleground should only start when there are full groups on both sides.

this would at the same time also reduce the impact of the second bug.

Further, i would propose to introduce the daily random battleground which should unlock at level 80, already in the current bracket 70, setting it to a minimum of 20 players required (10 per teamside) (potentially with adjusted honor rewards). This would allow for
1. more incentives for people to play battleground
2. more than just warsong being played outside of call to arms

### Source

.

### Steps to reproduce the problem

.

### Extra Notes

Arguments that there arent enough players doing pvp may hold true for low level brackets, but for the currently highest brackets, we see up to two alteracs running in parallel, resulting in above 110 players. there is simply not enough playerbase that is pvp focused only, and no incentive for non-pvp focused players

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/642bd1936b341a098d7c8d5f3089988409e4d6bb

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by chauusie** [comment]
Of course! I am just asking if you still find the problem, and if you would like to discuss it then the discussion board would be a good place? Otherwise, I can close the ticket. Unless there is a continued problem currently with this in BGs?

I don't PVP a lot, but games tend to be even when I do, the odd time they aren't seem to be people not accepting the queue pop. However if you find there still a large problem I can keep it open and we can try to triage it somewhat
